public_html/app/controllers/Users.php

```diff
@@ -0,0 +1,190 @@
+<?php
+
+    class Users extends Controller {
+        public function __construct() {
+            $this->userModel = $this->model('User');
+        }
+
+
+        // REGISTER
+        public function register() {
+            if ($_SERVER['REQUEST_METHOD'] == 'POST') {
```

Comment:
Dont repeat yourself. Create functions for this.

Reply:
1. Should this function be at folder tools or helpers?